cmd/erasure-multipart.go

```diff
@@ -481,7 +528,7 @@ func (er erasureObjects) PutObjectPart(ctx context.Context, bucket, object, uplo
 	var online int
 	defer func() {
 		if online != len(onlineDisks) {
-			er.deleteObject(context.Background(), minioMetaTmpBucket, tmpPart, writeQuorum)
+			er.renameAll(context.Background(), minioMetaTmpBucket, tmpPart)
```

Comment:
I thought we have moveToTrash now, wouldn't that work ?

Reply:
renameAll() moves to trash @vadmeste